Library/RemoteConfig/RemoteConfligClient.swift

```diff
@@ -0,0 +1,32 @@
+import FirebaseRemoteConfig
+
+public class RemoteConfigClient: RemoteConfigClientType {
```

Comment:
...this file! Which is simply a wrapper class around our `RemoteConfig.remoteConfig()` ObjC object.
Notice we hide the actual object as a `private var` and then expose the functions through our protocol.
FYI: this works because its' a _Swift_ class not an _ObjC_ class.

So let's go ahead and test this class with out `MockRemoteConfigClient`.

Reply:
ohh makes total sense!